eth2.0-implementers-calls/call_059.md

```diff
@@ -0,0 +1,212 @@
+
+**Protolambda** - The rustest PR opens in October last year, but introduced compression to the test formats. So that is going to take as much as like 10 or more gigabytes of this data, but rather like half a gigabytes, so you're tests would have to go through this compression step that aside from that. It's an improvement in the like a large improvement in the test size and then later on we had this issue opens about further improving that those test formats everyone kind of agreed on compression. So we're leaving it at that's right now. Maybe in the future we'll have a more sophisticated customers with deduplication. But right now this seems like the better and more direct option to reduce the test size for the next hard work.
+
+**Danny** - great in the numbers ran in October I think werre reducing the outputs from 11 gigabytes like half a gigabyte that's probably changed now with an additional Fork that still very significant. And this just adds in dot SOC Snappy extension rather than just SOC and we also removed the dot yaml versions of states and operations, which from our understanding and investigation in October there weren't really used by anybody. Okay, and showei what's the status on fork choice tests? Are we going to have maybe a sample of that coming out in the pre-release. 
+
+**Hsiao-Wei** - Yes. So currently the Piat 2202 has shown with a preview of the incoming test format and it's follow-up of Axic's previous work. I think it was discussed on the to state tests repo. So there are some differences between the previous proposal and this proposal. Mainly it's for making it more easy for high-stake to generate the test. So you can if you want to see the test vectors there example of the the get hat test and you can check the states file, which is it's kind of like a our beta configuration, but it shows us other states that the client has to check and checks is basically like the assertions like you have to check the value if it's equal to the test vectors and we have the block and attestations files in the test vector output so currently I use just attestation and participation has roots to present the file. But I'm thinking I can maybe just use the first eight bytes or like four bytes to make the filename shorter. Yeah. So if you if you have time, you can check the test vectors up to 202 and see if it would be like some drawbacks in this PR right now.And if there's no further issues, then I will first compute to get testnet for clients to try out.
```

Comment:
Another Alex 😄

```suggestion
**Hsiao-Wei** - Yes. So currently the PR 2202 has shown with a preview of the incoming test format and it's follow-up of Alex Vlasov's previous work. I think it was discussed on the to state tests repo. So there are some differences between the previous proposal and this proposal. Mainly it's for making it more easy for high-stake to generate the test. So you can if you want to see the test vectors there example of the the get hat test and you can check the states file, which is it's kind of like a our beta configuration, but it shows us other states that the client has to check and checks is basically like the assertions like you have to check the value if it's equal to the test vectors and we have the block and attestations files in the test vector output so currently I use just attestation and participation has roots to present the file. But I'm thinking I can maybe just use the first eight bytes or like four bytes to make the filename shorter. Yeah. So if you if you have time, you can check the test vectors up to 202 and see if it would be like some drawbacks in this PR right now. And if there's no further issues, then I will first compute to get testnet for clients to try out.
```